### ISSUE TITLE
isort: skip not being respected in 5.x.x

### ISSUE THREAD

**Issue by Gricha** [timothycrosley/isort#1337]
Hey! I've found another issue I'm battling, the last one that I think is stopping me from adopting isort 5 fully. Find attached compressed example module that presents the issue
[example_module.zip](https://github.com/timothycrosley/isort/files/4949004/example_module.zip)

The idea here is that `isort: skip` seems to be ignored. If you look at `a.py`, it does a couple of things - the ignored import imports a specific module that has to be in a specific order. Right now, testing on isort 5.1.4, this import will be thrown to the bottom. (the zip has my `.isort.cfg`, it's profiling for `black`).

Now interesting observation I had is that if I don't have a global import at the top (`import logging`) that issue does not come up.

Please let me know if I could help here!

**Comment by timothycrosley** [comment]
Hi @Gricha,

TLDR: try the [`# isort: split`](https://timothycrosley.github.io/isort/docs/configuration/action_comments/#isort-split) action comment and see if it works for you

For better or worse this is the intended and [documented](https://timothycrosley.github.io/isort/docs/configuration/action_comments/#isort-skip) behaviour of the `# isort: skip`  action comment. The reason why the behaviour is confusing, I believe, really comes down to that the idea of "skipping" one import in a series of imports is confusing. Here's an example:

```
import c
import b # isort: skip
import a
``` 
What is isort to do here? If the import is at the top, isort can ignore it entirely, once it's in the middle there is no clearly "correct" way to skip. isort: skip has the behaviour of putting it into the bottom. So that you can at least explicitly group your skipped import dependencies. For example to keep the above working if a depended on b no matter how many other imports you added you could do:

```
import c
import b # isort: skip
import a # isort: skip depends on b
``` 

However, isort 5.0.0 adds a new action comment `isort: split` that provides another way of keeping imports in-place. Here isort keeps all imports that take place above it above it, and all imports that place below it below it:

```
import c
import b `# isort: split
import a
```

Hopefully this is helpful! Let me know if this solution works for you :)

Thanks!

~Timothy

**Comment by Gricha** [comment]
Oh that completely makes sense! I didn't know about existence of `split` - it worked perfectly, thank you!
